Healthy/Classes/Modules/Dashboard/Views/FoodTagsView/Layout/UICollectionViewLayout+Extension.swift

```diff
@@ -0,0 +1,24 @@
+import UIKit
+
+extension UICollectionViewLayout {
+
+    static func createTagsLayout() -> UICollectionViewCompositionalLayout {
+
+        let itemSize = NSCollectionLayoutSize(
+            widthDimension: .estimated(100),
+            heightDimension: .fractionalHeight(1.0))
+        let item = NSCollectionLayoutItem(layoutSize: itemSize)
+
+        let groupSize = NSCollectionLayoutSize(
+            widthDimension: .estimated(100),
+            heightDimension: .fractionalHeight(1.0))
+        let group = NSCollectionLayoutGroup.horizontal(layoutSize: groupSize, subitems: [item])
+
+        let section = NSCollectionLayoutSection(group: group)
+        section.orthogonalScrollingBehavior = .continuousGroupLeadingBoundary
+        section.contentInsets = NSDirectionalEdgeInsets(top: 8, leading: 25, bottom: 8, trailing: 5)
```

Comment:
```suggestion
        section.contentInsets = NSDirectionalEdgeInsets(top: 8, leading: 20, bottom: 8, trailing: 20)
```

Leading should match trailing unless the design states another case.